### PR TITLE
Added scripts to run Gaia and IBC relayer (#79)

### LINKED ISSUE #79: Add instructions for running locally with gaia and relayer

### PR DESCRIPTION
**Pull request by shanev**
Fixes #79.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -78,3 +78,6 @@ proto-check-breaking:
 	@buf check breaking --against-input '.git#branch=master'
 
 .PHONY: proto-all proto-gen proto-lint proto-check-breaking
+
+ci-sign: 
+	drone sign public-awesome/stakebird --save
```
Comment (shanev) [inline Makefile:83]: 🤘

File: app/app.go [hunk 2]
```diff
@@ -376,7 +376,7 @@ func NewRocketApp(
 	// CanWithdrawInvariant invariant.
 	app.mm.SetOrderBeginBlockers(
 		upgrade.ModuleName, mint.ModuleName, curating.ModuleName, distr.ModuleName,
-		slashing.ModuleName, evidence.ModuleName, ibc.ModuleName,
+		slashing.ModuleName, evidence.ModuleName, staking.ModuleName, ibc.ModuleName,
```
Comment (shanev) [inline app/app.go:379]: 😬, don't `git blame` lol

File: contrib/ibc/stakebird-xfer.sh [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+rly lite init ibc0 -f
+rly lite init ibc1 -f
+
+# At this point the relayer --home directory is ready for normal operations between
+# ibc0 and ibc1. Looking at the folder structure of the relayer at this point is helpful
+tree ~/.relayer
+
+# Now you can connect the two chains with one command:
+# rly -d tx link demo
```
Comment (shanev) [inline contrib/ibc/stakebird-xfer.sh:46]: Should be able to uncomment now.

Comment (jhernandezb) [reply contrib/ibc/stakebird-xfer.sh]: Done!

**Pull request merged**